Composer/packages/server/src/services/storage.ts

```diff
@@ -68,6 +68,20 @@ class StorageService {
   // return connent for file
   // return children for dir
   public getBlob = async (storageId: string, filePath: string) => {
+    if (filePath === 'This PC') {
+      const diskNames = ['C:/', 'D:/', 'E:/', 'F:/', 'G:/', 'H:/', 'I:/', 'J:/', 'K:/'].filter(n => fs.existsSync(n));
```

Comment:
have you tested the behavior if I rename 'This PC' to another name?

Reply:
![image](https://user-images.githubusercontent.com/8528761/76979897-8adb3a00-6973-11ea-9f7f-e9063796fc06.png)

Reply:
This PC is a sample name. I think we just need a way for users to switch disk here.

Reply:
At least we need to verify 2 scenarios to know your code's limitation:
1. does it work after user renamed the PC name?
2. does it work for users in different regions whose system language is not English?

Even if we don't care about these two scenarios, the string 'This PC' shouldn't be hard-coded and appears in two different files.

Reply:
I updated code to take scenarios 2 into consideration and translated 'this PC'. It should be good now.